gulpfile.coffee

```diff
@@ -62,7 +64,12 @@ paths =
       "#{dir_style}/style.less"
     ]
   script: [
-      "#{dir_script}/**/*.coffee"
+      "#{dir_build}/script/common/api.js"
+      "#{dir_build}/script/common/util.js"
+      "#{dir_build}/script/site/admin.js"
+      "#{dir_build}/script/site/auth.js"
+      "#{dir_build}/script/site/user.js"
+      "#{dir_build}/script/site/app.js"
```

Comment:
The only thing that confuses me - it is a necessity to register scripts manually.